app/controllers/smart_answers_controller.rb

```diff
@@ -109,7 +109,7 @@ def set_expiry(duration = 30.minutes)
   end
 
   def setup_navigation_helpers_and_content_item
-    @content_item = Services.content_store.content_item!("/" + params[:id]).to_hash
+    @content_item = Services.content_store.content_item("/" + params[:id]).to_hash
```

Comment:
Is this a change to the API, or are we intentionally changing which method we call? If so, why?